### PR TITLE
fix(dapi): tenovi wh send to all connected users + revoke token for t…

### PR DESCRIPTION
**Pull request by RamilGaripov**
…est device ID

refs. metriport/metriport#849

### Description

- Send WH for each connectedUser associated with a device ID
- Fixed the edge case where revoke token and delete device were broken for Tenovi's test device ID. 

### Release Plan

- Nothing special

### CODE REVIEW

File: packages/api/src/providers/tenovi.ts [hunk 1]
```diff
@@ -91,11 +92,13 @@ export class Tenovi extends Provider {
       const url = `${Tenovi.URL}/${Tenovi.API_PATH}/hwi/unlink-gateway/${deviceId}/`;
 
       try {
-        await axios.get(url, {
-          headers: {
-            Authorization: `Api-Key ${Tenovi.apiKey}`,
-          },
-        });
+        if (deviceId !== TENOVI_TEST_DEVICE_ID) {
```
Comment (leite08) [inline packages/api/src/providers/tenovi.ts:95]: NABD, but it would look cleaner to have the conditional right at the start of the function, so we don't execute anything if its a test device.

Comment (RamilGaripov) [reply packages/api/src/providers/tenovi.ts:95]: The logic here is to not execute the axios request to de-activate the test device on Tenovi servers. But we still want to remove the test device from the user's providerMap. Although some refactoring is possible to not generate the URL.

Comment (leite08) [reply packages/api/src/providers/tenovi.ts]: yeah, 💯 

I wrongly assumed all we did there was the call to the external API, good call

File: packages/api/src/command/webhook/webhook.ts
Comment (leite08) [inline packages/api/src/command/webhook/webhook.ts:1]: weird why this is showing as a change, it's on `develop` already

**Review by leite08 — APPROVED** [review]

File: packages/api/src/command/webhook/webhook.ts
Comment (RamilGaripov) [reply packages/api/src/command/webhook/webhook.ts:1]: Yea. I noticed it, and not sure how that happened...
